app/api/admin/projects/[id]/route.ts

```
@@ -0,0 +1,161 @@
+
+    const db = await getDatabase()
+    const project = await db.collection<Project>('projects').findOne({
+      _id: new ObjectId(id)
+    })
+
+    if (!project) {
+      return NextResponse.json({ error: 'Project not found' }, { status: 404 })
+    }
+
+    // Build update object
+    const updateData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)